x-pack/plugins/index_management/public/application/services/documentation.ts

```diff
@@ -70,7 +70,7 @@ class DocumentationService {
 
     this.dataStreams = links.elasticsearch.dataStreams;
     this.esDocsBase = links.elasticsearch.docsBase;
-    this.enrichPolicies = links.elasticsearch.enrichApis;
```

Comment:
is `enrichApis` being used elsewhere in the code still? Or can it be removed from `get_doc_links.ts`?